.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!--
+🚨HOLD UP PLEASE, BEFORE DELETING THIS TEMPLATE!🚨😁
+Thank you for contributing and please follow this guide before creating an issue! 🙌
+
+1. Do you have a usage question?
+=============================
+- Look for prior or closed issues (but please avoid replying to them if they're old)
+- Check the docs: https://www.styled-components.com/docs
+- Look for/ask questions on stack overflow: https://stackoverflow.com/questions/ask?tags=styled-components
+- Start a thread on our Spectrum help channel: https://spectrum.chat/styled-components/help
+
+2. Think you found a bug?
+=============================
+- Consider submitting a PR with a failing test instead
+- Use the "BUG TEMPLATE" below to report a test
+- Don't forget to 👉PROVIDE A REPRODUCTION PLEASE😉 (Codesandbox/Webpackbin links are provided, repositories are ok)
+- If you can't provide a reproduction, snippets of code can help, but are incomplete reports
+
+3. Do you have a feature request?
+=============================
+- Look for old & closed issues (replying might be ok if they're not too old or have no conclusion)
+- Otherwise: Remove the template below and provide thoughtful commentary *and code samples* on what this feature means for your product
+
+What will it allow you to do that you can't do today?
+How will it make current work-arounds straightforward?
+What potential bugs and edge cases does it help to avoid?
+Please keep it product-centric.
+-->
+
+<!-- BUG TEMPLATE -->
+## Environment
+<!-- Please add your stylelint configurations to the `npmPackages` below, run this command inside your project and paste its contents here (it automatically copies to your clipboard) -->
+`npx envinfo --system --binaries --npmPackages styled-components,babel-plugin-styled-components,stylelint-processor-styled-components, --markdown --clipboard`
```

Comment:
Let's add packages mentioned in README.md and how about removing `babel-plugin-styled-components`? Then it is, `--npmPackages stylelint, styled-components, stylelint-processor-styled-components, stylelint-config-styled-components, stylelint-config-recommended`.

And of course, `.stylelintrc` is welcome to be provided.

Reply:
Good point!